cleanup_utils/integration_cleanup.py

```diff
@@ -103,7 +116,9 @@ def run_integration_cleanup(dry_run=False):
     )
     # Run DNS cleanup erasing all integration entries except for those on
     # the deletion_blacklist
-    hashes_to_clean = aws_cleanup.cleaned_up_hashes + os_cleanup.cleaned_up_hashes
+    hashes_to_clean = (
+        aws_cleanup.cleaned_up_hashes + os_cleanup.cleaned_up_hashes +
+        mysql_cleanup.cleaned_up_hashes)
```

Comment:
@jamestait, similar to a previous comment, it is a good idea to move the closing parenthesis to a new line.

```python
hashes_to_clean = (
    aws_cleanup.cleaned_up_hashes + os_cleanup.cleaned_up_hashes +
    mysql_cleanup.cleaned_up_hashes
)
```